app/Resources/views/facebook/index.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html.twig' %}
+
+{% block content %}
+    <main class="facebook">
+        {% include 'facebook/header.html.twig' %}
```

Comment:
Use the `include()` function instead please to be consistent with all templates.